projects/common/src/color/color-palette.test.ts

```diff
@@ -31,4 +31,26 @@ describe('Color palette', () => {
     expect(() => new ColorPalette(['black'])).toThrow();
     expect(() => new ColorPalette(['white', 'black'])).not.toThrow();
   });
+
+  test('should return color combinations correctly', () => {
+    const palette = new ColorPalette(['#fffbeb', '#140300']);
+    expect(palette.getColorCombinations(2)).toEqual([
+      {
+        background: 'rgb(255, 251, 235)',
+        foreground: '#080909'
+      },
+      {
+        background: 'rgb(20, 3, 0)',
+        foreground: '#FFFFFF'
+      }
+    ]);
+  });
+
+  test('should generate color for a string as expected from a limited set', () => {
+    const palette = new ColorPalette(['#fffbeb', '#140300', '#789ab7']);
+    expect(palette.getColorCombinationForId(2, 'test')).toEqual({
```

Comment:
Looking at how this is used, we could probably make the count optional and the second param. so just `getColorCombinationForString('test')`

Reply:
makes sense. updated @aaron-steinfeld